cmd/daytona/config/const.go

```diff
@@ -84,6 +85,8 @@ func GetDocsLinkFromGitProvider(providerId string) string {
 		return "https://learn.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate?view=azure-devops&tabs=Windows#create-a-pat"
 	case "aws-codecommit":
 		return "https://docs.aws.amazon.com/codecommit/latest/userguide/setting-up-gc.html and to configure AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY & AWS_DEFAULT_REGION read https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html"
+	case "gogs":
+		return "https://gist.github.com/tarunrajput/b506f0c607b06efaa3084503fdac428e"
```

Comment:
There is no documentation on the Gogs website for creating a PAT, so I made a gist mentioning the requird steps.

Reply:
It would be great if you could add any entry to https://github.com/daytonaio/docs about this and then link to that.

Reply:
done. https://github.com/daytonaio/docs/pull/179